src/main/java/co/org/osso/api/BookService.java

```diff
@@ -0,0 +1,52 @@
+package co.org.osso.api;
+
+import org.springframework.stereotype.Service;
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+public class BookService {
+
+    Map<String, Book> mapBooks = new HashMap<>();
```

Comment:
This field should be `private final` as it is never reassigned.
The field below can also be private, but not final.
Maybe rename the variable to `bookCache`, as that's effectively what it is.
```suggestion
    private final Map<String, Book> bookCache = new HashMap<>();
```

Reply:
Done